packages/swapper/src/swappers/thorchain/utils/getTradeRate/getTradeRate.ts

```diff
@@ -37,6 +38,13 @@ export const getTradeRate = async (
   sellAmount: string,
```

Comment:
Just an aesthetic thing - I see that only the buy asset ID is used in this function, but maybe it would look nicer to pass in `sellAsset: Asset, buyAsset: Asset` so that the arguments are of the same type.

Reply:
I see your point but im not changing it